src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1108,7 +1109,7 @@ public class CaseData extends Et1CaseData {
     @JsonProperty("resTseTextBox12")
     private String resTseTextBox12;
     @JsonProperty("resTseCopyToOtherPartyYesOrNo")
-    private String resTseCopyToOtherPartyYesOrNo;
```

Comment:
Is this field in prod? Looks like yes, then there will be validation errors if we change the datatype now. May be add a new field for DynamicListType?

Reply:
Data type will stay the same as the code value Yes or No. That is why It will not affect the old cases.

Reply:
As discussed on slack, It will still have value as Yes or No
Just the labels will change.
Code will stay the same but Labels will be updated for User Interfaces to have html codes
rendered while showing the radio buttons. (

Reply:
This should be a new field, the data structure of a String and dynamiclist is completely different

Reply:
Will implement it as a new field. Thanks Harpreet.